week-1/exercises/paths.js

```diff
@@ -12,7 +12,11 @@
 const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
 const lastSlashIndex = filePath.lastIndexOf("/");
 const base = filePath.slice(lastSlashIndex + 1);
-console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
-// Create a variable to store the ext part of the variable
+const dir = filePath.slice(filePath.indexOf("/"), filePath.lastIndexOf("/"));
```

Comment:
You've got to the correct answer here. Could you reuse the "lastSlashIndex" variable declared above?